src/transformers/baseSourceMapTransformer.ts

```diff
@@ -120,6 +122,11 @@ export class BaseSourceMapTransformer {
 
                 this._authoredPathsToMappedBPs.set(argsPath, args.breakpoints);
 
+                // Store the client breakpoint Ids for the mapped BPs as well
+                if (ids) {
+                    this._authoredPathsToClientBreakpointIds.set(argsPath, ids);
```

Comment:
Does this need to be in a map on the instance? Since they aren't referenced outside of this method.